UICatalog/Scenarios/SyntaxHighlighting.cs

```diff
@@ -282,8 +282,8 @@ private void ApplySyntaxHighlighting ()
         };
 
         _textView.TextChanged += (s, e) => HighlightTextBasedOnKeywords ();
-        _textView.DrawContent += (s, e) => HighlightTextBasedOnKeywords ();
-        _textView.DrawContentComplete += (s, e) => HighlightTextBasedOnKeywords ();
+        _textView.DrawingContent += (s, e) => HighlightTextBasedOnKeywords ();
```

Comment:
The color is only rendered when we click in the `TextView` on open.

Reply:
Does this occur in v2_develop?

Reply:
> Does this occur in v2_develop?

No.

Reply:
Fixed.